src/http3eventloop.cc

```diff
@@ -295,6 +295,33 @@ namespace quic
       progress_->Send(&report, 1);
   }
 
+  void Http3EventLoop::informHttp3ServerListening(Http3Server *serverobj)
```

Comment:
Two things, please move it up where the other Http3Server callbacks are, they are actually sorted a bit...
Second, merge the three functions into `informHttp3ServerStatus` with an enum indicating the purpose and maybe some data. There is actually also a mechanism that allows passing memory that is later deleted on the other thread. This way also extended status information can be added to the object passed to the js side. (Also changes in napi may require editing all handlers, so I try to keep them at a minimum....)